src/capwap_loc_provider.erl

```diff
@@ -134,7 +134,7 @@ terminate(Reason, _) ->
 %% user api
 
 is_enabled() ->
-    case application:get_env(capwap, location_provider) of
+    case application:get_env(location_provider) of
```

Comment:
If you want use this approach - need also use same approach in https://github.com/travelping/capwap/blob/3.2.0-rc.2/src/capwap_loc_provider.erl#L137

Reply:
Weird, I thought this was pushed. Will check.

Reply:
This was added in the scope at https://github.com/travelping/capwap/pull/62 - what I was mean before - in two places you are use two different approaches to use `env`

Reply:
I'm using only the app-agnostic form. I'll check.

Reply:
You mean _the other calls_. Ok.

Reply:
Yes